tests/unit/docs/__init__.py

```diff
@@ -135,6 +135,30 @@ def _setup_models(self):
             }
         }
 
+        self.examples_json_model = {
```

Comment:
I'd prefer this just move to the test module where it's used.  Otherwise you have to keep jumping between both files to understand the test.